### PR TITLE
Feature/adding ideas

### PR DESCRIPTION
**Pull request by Denzel-Turambi**
This adds the query selectors for the buttons and form elements. It will also add an event listener and function for the save button.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1 +1,27 @@
-var ideas = [];
\ No newline at end of file
+var Ideas = require('../Idea.js');
+var ideas = [];
+
+//query selector go here:
```
Comment (asutherland91) [inline main.js:4]: Good idea labeling the sections!

Comment (asutherland91) [inline main.js:4]: I like the naming conventions of the query selectors. Really easy to understand

**Review by brendel-r — COMMENTED** [review]
It looks like we are heading in the right direction. Keeping our code dry is key!!

**Review by asutherland91 — COMMENTED** [review]
I think it is a great start thus far. Lets keep up the good work

**Review by RandyGitProjects — COMMENTED** [review]
Looks good getting all query selectors ready. Will eventually need more event listeners.

**Pull request merged**